opamp/observiq/observiq_client.go

```diff
@@ -210,6 +210,13 @@ func (c *Client) Connect(ctx context.Context) error {
 			// SaveRemoteConfigStatusFunc
 		},
 		PackagesStateProvider: c.packagesStateProvider,
+		Capabilities: protobufs.AgentCapabilities_AgentCapabilities_ReportsStatus |
```

Comment:
I realized we haven't gotten to tests yet but can we make sure we add a check in the `TestNewClient` test that the capabilities match what we expect?